### DISCUSSION TITLE
Themes on Ipad/ios?

### DISCUSSION THREAD

**Discussion by HatfieldS** [AnubisNekhet/AnuPpuccin#295] [Snippets]
Hello! So to preface, I know Ipad is a brat, and I'm well aware it may not be possible.

Trying to get this theme to work on my ipad. I've used a program called a-Shell to access Obsidian's directory, add a .obsidian/themes file, and then pull the github repository into there. But it doesn't seem to change anything? I may have wasted the past 2ish hours, but I've at least learned some neat things, so not a total waste if it won't work.

Thank you!

**Comment by DigiRhys** [comment]
I found a solution via reddit comment a few weeks ago to load snippets onto my iPad! It's still a bit of a pain, but it has worked without issue so far :0

The comment with steps is [here](https://www.reddit.com/r/ObsidianMD/comments/1305ozs/comment/jpf56l2/), but putting the steps directly here as well:
1. Download Toolbox Pro (can confirm it is a free app, don't worry!)

2. Open Toolbox Pro -> Settings -> Folder Bookmarks and click on the + icon on top right

3. Choose the Obsidian vault in your Obsidian folder where you want to apply the CSS snippets and press Open (e.g. I made two Bookmarks since I have two vaults On My iPad -> Obsidian -> Mine and one On My iPad -> Obsidian -> D&D. I nicknamed each bookmark so I know which vault it's for, I recommend doing the same if you're setting this up for multiple vaults too)

4. Click save bookmarks

5. Save your CSS file and **turn it into a txt file first**! This is important or the Shortcut won't work (I forget sometimes to change em from css to txt first lol you can rename directly from the native iOS Files app, make sure you have it set to show file extensions. It'll warn you about changing the extension and make you confirm, it's fine)

6. Download [this shortcut](https://www.icloud.com/shortcuts/3df968e8ee1d4d52b1d17cffa319fc1b) (https://www.icloud.com/shortcuts/3df968e8ee1d4d52b1d17cffa319fc1b) by ryanjamurphy (he confirms [here](https://forum.obsidian.md/t/apple-shortcuts-share-your-ideas/15149/33) you can select multiple txt files to add at once👍🏻)

7. Open the Shortcuts app to view and edit the new Save CSS Snippets Shortcut, you'll need to direct it to save to the vault folder you created a boomark for Tool Box Pro earlier. This should be step 6 in the shortcut, the second-to-last step "Save Renamed Item to Obsidian Sync directory"

8. Select "Obsidian Sync directory" to select the bookmarked folder. If you have multiple vaults you want to add snippets to, you'll have to make unique copies of this shortcut to direct each one to the correct vault folder bookmark

9. That's it! Run the shortcut & select the snippets you changed to txt files, and all should be working!
